src/GRA/SiteSettingDefinitions.cs

```diff
@@ -120,13 +120,27 @@ public static readonly Dictionary<string, SiteSettingDefinition> DefinitionDicti
                     Category = typeof(SiteSettingKey.Users).Name,
                     Format = SiteSettingFormat.Boolean,
                 },
+                [SiteSettingKey.Triggers.DisallowTriggersBelowPoints] = new SiteSettingDefinition
+                {
+                    Name = "Disallow triggers below these points",
```

Comment:
This text should probably be "Disallow triggers which activate at point values below the specified number of points"